deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -625,6 +626,279 @@ void setGlConstants(T &proto) {
 
   // external
   JS_GL_SET_CONSTANT("TEXTURE_EXTERNAL_OES", 0x8D65);
+  
```

Comment:
Maybe it makes sense to delete the WebGL constants and just use this list for all of them?